sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/samples/java/com/azure/messaging/eventhubs/checkpointstore/blob/EventProcessorBlobCheckpointStoreSample.java

```diff
@@ -24,15 +24,15 @@
     private static final String SAS_TOKEN_STRING = "";
     private static final String STORAGE_CONNECTION_STRING = "";
 
-    public static final Consumer<PartitionEvent> PARTITION_PROCESSOR = partitionEvent -> {
+    public static final Consumer<EventContext> PARTITION_PROCESSOR = eventContext -> {
         System.out.printf("Processing event from partition %s with sequence number %d %n",
-            partitionEvent.getPartitionContext().getPartitionId(), partitionEvent.getData().getSequenceNumber());
-        if (partitionEvent.getData().getSequenceNumber() % 10 == 0) {
-            partitionEvent.getPartitionContext().updateCheckpoint(partitionEvent.getData()).subscribe();
+            eventContext.getPartitionContext().getPartitionId(), eventContext.getEventData().getSequenceNumber());
+        if (eventContext.getEventData().getSequenceNumber() % 10 == 0) {
+            eventContext.updateCheckpoint();
         }
     };
 
-    public static final Consumer<EventProcessingErrorContext> ERROR_HANDLER = errorContext -> {
+    public static final Consumer<ErrorContext> ERROR_HANDLER = errorContext -> {
```

Comment:
In azure-core-amqp, we have AmqpErrorContext, that is a field within an AmqpException. I wonder if there will be confusion here because the names are so similar.

Reply:
It was one of Ramya's comment to make all EP contexts to either have prefix or not have prefix. InitializationContext, CloseContext, EventContext, ErrorContext all should either have a prefix or not. We can finalize the names next week.